engine/src/bmr/BlazingMemoryResource.h

```diff
@@ -171,7 +171,7 @@ class blazing_device_memory_resource : public BlazingMemoryResource {
    * 
    * @param[in] options Optional options to set
    * ----------------------------------------------------------------------**/
-    void initialize(const rmmOptions_t *new_options) {
+    void initialize(const rmmOptions_t *new_options, float BLAZING_DEVICE_MEM_RESOURCE_CONSUMPTION_THRESHOLD) {
```

Comment:
does the fact that this is some config merit an all caps deviation from normal parameter naming?

Reply:
Constants are traditionally set as ALL CAPS. We have used that pattern for all configurable parameters in the past. This is sticking with our current naming convention for config parameters which as has been to set them as all caps.

Reply:
This isnt a compile time constant. It is actually part of the state of the context.